htmresearch/algorithms/column_pooler.py

```diff
@@ -543,7 +543,8 @@ def _rightVecSumAtNZGtThreshold_sparse(sparseMatrix,
   an eventual C implementation would.
   """
   denseArray = numpy.zeros(sparseMatrix.nCols(), dtype=GetNTAReal())
-  denseArray[sparseBinaryArray] = 1
+  if len(sparseBinaryArray) > 0:
```

Comment:
(Ignore my second comment -- your code solves that problem. But `np.asarray` would be good.)